molecule/provisioner/ansible/plugins/libraries/README.rst

```diff
@@ -0,0 +1,11 @@
+Ansible Module overrides for Molecule
```

Comment:
Please do this in the Sphinx documentation as well. 

Will save us time for tickets later on ...

Reply:
Not sure if I need to document the fact internally we do not use same ansible modules. Do we? Or you refer to another part of the documentation.

Reply:
Ah, I get you. I thought that this will be possible to change through some experimental end-user intervention. If that is not the case then yes, let's avoid documenting this.